src/RVRP_solution.jl

```diff
@@ -3,23 +3,23 @@
 mutable struct Action
     id::String
     location_id::String
-    operation_type::Int # 0- Depot, 1 - Pickup, 2 - Delivery, 3 - otherOperation # should be picked quantity
     request_id::String # if any request is linked to a request the operation
-    scheduled_start_time::Float64 # post-compute
```

Comment:
I am going to revert the deletion of this line because we actually need to use the scheduled time from the solver. In some complex problems it may be difficult to retrieve the best scheduled time just by looking at the sequence.